pkg/query-service/app/querier/v2/helper.go

```diff
@@ -0,0 +1,306 @@
+package v2
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+	"sync"
+	"time"
+
+	logsV3 "go.signoz.io/signoz/pkg/query-service/app/logs/v3"
```

Comment:
will logs and traces continue to use `v3` even in `querier/v2`?

Reply:
Yes, they will continue to use v3.